README.md

```diff
@@ -492,11 +493,11 @@ Notable missing features include:
 - `[AllowShared]`
 - `[Default]` (for `toJSON()` operations)
 - `[Global]`'s various consequences, including the named properties object and `[[SetPrototypeOf]]`
-- `[LenientSetter]`
-- `[LenientThis]`
```

Comment:
I seem to remember `LenientThis` was not moved to the list of implemented features because it was implemented in some places but not others. I can't find anything now though so I suppose it's since been implemented.